ts/components/messages/MessageDetail/PdfViewer.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import { StyleSheet } from "react-native";
+import Pdf from "react-native-pdf";
+import LoadingSpinnerOverlay from "../../LoadingSpinnerOverlay";
+import { IOColors } from "../../core/variables/IOColors";
+import I18n from "../../../i18n";
+
+const styles = StyleSheet.create({
+  pdf: {
+    flex: 1,
+    backgroundColor: IOColors.bluegrey
+  }
+});
+
+type OwnProps = {
+  downloadPath: string;
+};
+
+type Props = OwnProps &
+  Omit<React.ComponentProps<typeof Pdf>, "source" | "style">;
```

Comment:
Why are we omitting the style property? Wouldn't it be better to merge the input one with the internal we are using, giving a future developer the option to set her own style?